### ISSUE TITLE
"Play" message in the console

### ISSUE THREAD

**Issue by oza-c** [farhanfadila1717/slide_countdown#10]
```
Visibility(
                          visible: _remoteConfig.getBool('*************'),
                            child: const SlideCountdown(
                         duration: Duration(days: 1),
                     )),
```

When the app is opened, a message is displayed in the console ("Play").
Small error? or wanted? Print call cannot be localized by me

Greetings :)

**Comment by farhanfadila1717** [comment]
Hi @oza-c 🙌, message `Play` from `StreamDuration` package when is initialize, i'm already upgrade lastest version  of `StreamDuration` package, you can try in `SlideCountdown` [ver0.2.4-dev.1 ](https://pub.dev/packages/slide_countdown/versions/0.2.4-dev.1)

**Comment by farhanfadila1717** [comment]
Fixed in [slide_countdown: ^0.2.4](https://pub.dev/packages/slide_countdown/versions/0.2.4)
